stories/core/ComboBox.stories.tsx

```diff
@@ -497,3 +497,24 @@ WithCustomMessageIcon.args = {
     <StatusMessage startIcon={<SvgCamera />}>This is a message</StatusMessage>
   ),
 };
+
+export const Virtualized: Story<Partial<ComboBoxProps<string>>> = (args) => {
```

Comment:
I missed the explanation of why the `WithMessage` story was removed? I think there should be a note about it in the PR